client/packages/requisitions/src/ResponseRequisition/ListView/ProgramRequisitionOptions.tsx

```diff
@@ -68,15 +68,18 @@ const useProgramRequisitionOptions = (
       disabled: false,
     },
     orderTypes: {
-      options: program?.customerAndOrderTypes.flatMap(c => c.orderTypes) || [],
+      options:
+        program?.customerAndOrderTypes
+          .filter(c => c.customer.id === customer?.id)
+          .flatMap(c => c.orderTypes) || [],
       value: orderType,
       set: setOrderType,
-      disabled: program === null,
+      disabled: program === null || customer === null,
```

Comment:
Ah great! I had wondered before if it was intentional to be able to select order type before customer :D

Reply:
The option is just mapped before, but you can't select orderType before customer XD